docs/source/benchmarking/materials_science_and_engineering/oiesoftware.rst

```diff
@@ -1,11 +1,11 @@
 Open Innovation Environment (OIE) domain ontologies, Software module (OIESoftware)
```

Comment:
convert `Open Innovation Environment (OIE) domain ontologies, Software module (OIESoftware)` to `Open Innovation Environment Software (OIESoftware)`